core/src/main/java/de/jplag/TimeUtil.java

```diff
@@ -12,10 +14,8 @@ private TimeUtil() {
      * @return Readable representation of the time interval.
      */
     public static String formatDuration(long durationInMilliseconds) {
-        int timeInSeconds = (int) (durationInMilliseconds / 1000);
-        String hours = (timeInSeconds / 3600 > 0) ? (timeInSeconds / 3600) + " h " : "";
-        String minutes = (timeInSeconds / 60 > 0) ? ((timeInSeconds / 60) % 60) + " min " : "";
-        String seconds = (timeInSeconds % 60) + " sec";
-        return hours + minutes + seconds;
+        Duration duration = Duration.ofMillis(durationInMilliseconds);
+        return String.format("%dh %02dmin %02ds %03dms", duration.toHours(), duration.toMinutesPart(), duration.toSecondsPart(),
```

Comment:
This will always output `0h ...` if  the execution took less than an hour. The same with other time units. The old code did not do that. Is this the desired behavior?

Reply:
Yes, the new representation is chosen to be more consistent.